docs/3.x.x/en/getting-started/installation.md

```diff
@@ -5,8 +5,8 @@ Installation is very easy and only takes a few seconds.
 ## Requirements
 
 Please make sure your computer/server meets the following requirements:
- - [Node.js](https://nodejs.org) >= 9: Node.js is a server platform which runs JavaScript. Installation guide [here](https://nodejs.org/en/download/).
- - [MongoDB](https://www.mongodb.com/) >= 2.6: MongoDB is a powerful document store. Installation guide [here](https://www.mongodb.com/download-center?j#community).
+ - [Node.js](https://nodejs.org) >= 10.x: Node.js is a server platform which runs JavaScript. Installation guide [here](https://nodejs.org/en/download/).
+ - [MongoDB](https://www.mongodb.com/) >= 6.x: MongoDB is a powerful document store. Installation guide [here](https://www.mongodb.com/download-center?j#community).
```

Comment:
So... This is requiring using Mongo 6? 🤔 I think the latest version right now is 4.x, isn't? 🤔

Reply:
Sorry, that's my typo. It should be 3.x for MongoDB.
But this raises a new question: Do we need to include the requirement of npm and other databases (MySQL, MariaDB, PostgreSQL) on installation page?

Reply:
They should be defined yes @henrych4